client/reader/site-stream/index.jsx

```diff
@@ -127,10 +128,6 @@ const SiteStream = React.createClass( {
 			title = this.translate( 'Loading Site' );
```

Comment:
I'm not sure the reason why yet, but I'm noticing the title progresses differently on the Reader Discover screen between production and local:

![title](https://cloud.githubusercontent.com/assets/1779930/17596241/7751a0b8-5fbe-11e6-9c76-25b35a4ec8da.gif)

_Left: Development, Right: Production_

(Apologies for the long GIF, internet especially slow today 😆 )